themes/vue/source/css/_style-guide.styl

```diff
@@ -0,0 +1,82 @@
+$style-guide-bad-bg = lighten(desaturate($red, 80%), 80%)
+$style-guide-bad-text = darken(desaturate($red, 80%), 20%)
+$style-guide-good-bg = lighten(desaturate($green, 80%), 85%)
```

Comment:
It's kind of subjective, but personally I don't think the current background color is a bit too dark to indicate good. Maybe try something like `#dbf1d9`?

Reply:
We can tweak the colors after publishing, but I think this is good enough for now. The specific color you suggested looks very bright to me, like it's trying to highlight something rather than blending into the background. And since the rest of the page is using more subtle, desaturated colors, a more complete overhaul might be necessary to make something like that fit in.